onnxruntime/contrib_ops/cuda/bert/flash_attention/utils.h

```diff
@@ -0,0 +1,370 @@
+/******************************************************************************
+ * Copyright (c) 2023, Tri Dao.
+ ******************************************************************************/
```

Comment:
Who is Tri Dao?

Reply:
The author of [flash attention](https://github.com/Dao-AILab/flash-attention)

@aciddelgado, please update ThirdPartyNotices.txt to include it and [its license](https://github.com/Dao-AILab/flash-attention/blob/main/LICENSE).